plugins/scanners/scanoss/src/main/kotlin/ScanOss.kt

```diff
@@ -21,6 +21,11 @@ package org.ossreviewtoolkit.plugins.scanners.scanoss
 
 import com.scanoss.Scanner
 import com.scanoss.filters.FilterConfig
+import com.scanoss.settings.Bom
```

Comment:
Commit message:

```
The implementation converts ORT's SnippetChoices
into SCANOSS-specific rule types
```

Please do not wrap before column 75, and end sentences with a dot.